Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocValidationErrorIndicatorColumnRenderer.cs

```diff
@@ -65,6 +65,33 @@ protected override string GetAdditionalCssClassForDataCell (BocCellAttributeRend
       return CssClasses.DataCellValidationFailureIndicator;
     }
 
+    protected override void RenderTitleCellContents (BocColumnRenderingContext<BocValidationErrorIndicatorColumnDefinition> renderingContext)
```

Comment:
I want to try and avoid this extension point if we can do this via CSS. There's an idea/comment about replacing the `nbsp;` with a `span` and dummy width. This can then serve as the icon-target. 

Reason for avoiding the extension point is that making sure we cover "all" that is needed would be more effort than we should invest right now and doing it partially invites non-optimal usages downstream and added costs when we rework the extension point.

Having an icon for the title could perhaps work, doing it the same way as we do it with the title. (similiar properties, etc). 

TBD: do we perhaps want to a tooltip for the icon anyway? should the tooltip simply be the otherwise invisible column title?